src/miragevpn.mli

```diff
@@ -51,6 +51,10 @@ module Config : sig
         (** [connect-timeout] a.k.a. [server-poll-timeout]
         Timeout a connection attempt to a remote server after [SECONDS].
     *)
+    | Data_ciphers
+        : [ `AES_256_CBC | `AES_128_GCM | `AES_256_GCM | `CHACHA20_POLY1305 ]
+          list
+          k
```

Comment:
after looking into the openvpn man page and our implementation, I think that "cipher" should only allow the AES_256_CBC (in the configuration file), and data_ciphers should only allow the 3 AEAD ciphers in the configuration file.

the config is also used for the established session, where Cipher need to allow all ciphers (after NCP, the server will tell us which cipher to use, which may well be AEAD).

but also configurations without "cipher" should be valid (if tls-client/tls-server is used).

Reply:
done in 3403569